cobc/parser.y

```diff
@@ -886,6 +885,24 @@ check_relaxed_syntax (const cob_flags_t lev)
 	}
 }
 
+static void
+setup_default_collation (struct cb_program *program) {
+	switch (cb_default_colseq) {
+#ifdef COB_EBCDIC_MACHINE
+	case CB_COLSEQ_ASCII:
+#else
+	case CB_COLSEQ_EBCDIC:
+#endif
+		alphanumeric_collation = national_collation =
```

Comment:
Setting the national collation to the alphanumeric collation is likely wrong- but that was the case in some of the removed code paths too.
You may always set it to `NULL` for now and leave a TODO marker, or leave as is (hm a TODO/CHECKME marker would be good in any case)